src/data/index.ts

```diff
@@ -0,0 +1,69 @@
+import sqlite3, { Database } from 'better-sqlite3'
+import { existsSync, mkdirSync } from "fs"
+import initBotData from "./initializers/botData"
+import { Databases, Pragma } from './types'
+
+const rootDirectory = process.cwd()
+const dataDirectory = rootDirectory + "/src/data/database/"
+if (!existsSync(dataDirectory)) {
+    try {
+        mkdirSync(dataDirectory)
+    } catch (error) {
+        console.log(error)
+    }
+}
+
+const loadedDatabases: {
+    [key in Databases]?: Database
+} = {}
+
+const dbPragmas = {
+    [Pragma.DEFAULT]: "journal_mode = WAL"
```

Comment:
wat dis

Reply:
This is a pragma (setting) for the database that improves its speed drastically.
[sqlite docs](https://www.sqlite.org/wal.html)

Reply:
This is a pragma (setting) for the database that improves its speed drastically.
[sqlite docs](https://www.sqlite.org/wal.html)